packages/muelu/src/Utils/MueLu_Utilities_kokkos_decl.hpp

```diff
@@ -172,7 +172,7 @@ namespace MueLu {
 
     NOTE -- it's assumed that A has been fillComplete'd.
     */
-    static RCP<Vector> GetMatrixDiagonalInverse(const Matrix& A, Magnitude tol = TST::eps()*100); // FIXME
+    static RCP<Vector> GetMatrixDiagonalInverse(const Matrix& A, Magnitude tol = TST::eps()*100, const bool doLumped = false); // FIXME
```

Comment:
@jhux2 I'm not a big fan of the term "Lumped" or any reference to Lump when describing the rowSumAbs thing. I don't have a great alternative other than saying rowSumAbs. Lumping I feel refers to the moving of off diagonal entries (so zeroing out some entries) and grouping them with others (usually to the diagonal). This is what it means at least in the finite element community. The idea here is to produce a sparsified matrix with a similar effect as the original matrix. This is not really the same thing as taking the row sum of abs(a_ij). In this case, we are looking to scale the matrix (which can be thought of as a super cheap approximation of the matrix inverse). Normally, we use the diagonal and invert it ... but in this case we use the rowSumAbs thing.